bin/assembleCoadd.py

```diff
@@ -20,7 +20,34 @@
 # the GNU General Public License along with this program.  If not,
 # see <http://www.lsstcorp.org/LegalNotices/>.
 #
+from __future__ import print_function
 import lsst.pex.logging as pexLog
```

Comment:
`pexLog` is unused. You're not changing that line here, but maybe take this opportunity to remove it?